main.cpp

```diff
@@ -462,40 +460,40 @@ void Cinema::playFilm(bool, bool)
 {
     if(screenOn == true || projectorOn == true)
     {
-        std::cout<<"Quiet Please Movie is On"<<std::endl;
+        std::cout << "Quiet Please Movie is On" << std::endl;
     }
     else
     {
-        std::cout<<"Movie will Start soon"<<std::endl;
-    }
+        std::cout << "Movie will Start soon" << std::endl;
+    } 
 }
 
 void Cinema::setLightLevel(int)
```

Comment:
- [ ] why isn't your function argument being used?  You could assign its value to your member variable `lightLevel` before you start your `if/else if/else` logic.

Reply:
I've  changed it so it now changes the value of lightLevel